INSTALLATION.md

```diff
@@ -13,6 +13,52 @@ cilium install --version -service-mesh:v1.11.0-beta.1 --config enable-envoy-conf
 cilium hubble enable
 ```
 
+Alternatively, you can use Helm to install Cilium Service Mesh. For example, to install on GKE:
+
+```
+# Specify your cluster name and zone.
+CLUSTER_NAME=my-cluster
+CLUSTER_ZONE=us-west2-a
+
+NATIVE_CIDR="$(gcloud container clusters describe $CLUSTER_NAME --zone $CLUSTER_ZONE --format 'value(clusterIpv4Cidr)')"
+git clone git@github.com:cilium/cilium.git
+cd cilium
+git checkout origin/beta/service-mesh
+cat > service-mesh-values.yaml <<EOF
+image:
+  repository: quay.io/cilium/cilium-service-mesh
```

Comment:
For consistency would it work if this was `cilium/cilium`, but also uses the `suffix` setting like the operator image does at line 37?

Reply:
no the `suffix` setting is only available for operator image.